dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/command/QueuedCommand.java

```diff
@@ -41,7 +41,13 @@ public void cancel() {
 
     public void run(Channel channel) {
         if (channel.isActive()) {
-            channel.write(this, promise);
+            channel.write(this).addListener(future -> {
```

Comment:
The reason of changing is?

Reply:
The stream channel has not been constructed when the header/data is sent, the `ChannelPromise` of header/data comes from the `parent channel`